forest/cli/src/cli/snapshot_cmd.rs

```diff
@@ -316,13 +322,20 @@ fn clean(config: &Config, snapshot_dir: &Option<PathBuf>, force: bool) -> anyhow
     let snapshots_to_delete: Vec<_> = fs::read_dir(snapshot_dir)?
         .flatten()
         .map(|entry| entry.path())
-        .filter(|p| p.extension().unwrap_or_default() == "car")
+        .filter(|p| {
+            let ext = p.extension().unwrap_or_default();
+            ext == "car" || ext == "tmp"
```

Comment:
How about moving this predicate to a common method?

Reply:
Fixed